src/app/explorePools/page.tsx

```
@@ -558,22 +589,67 @@ function ExploreFatePoolsClient() {
+  // Clicking a column header re-sorts by that field; clicking the active field flips order.
+  const handleSort = useCallback((field: PoolSortField): void => {
+    setSortState(prev =>
+      prev.field === field
+        ? { field, order: prev.order === 'asc' ? 'desc' : 'asc' }
+        : { field, order: field === 'name' ? 'asc' : 'desc' }
+    );
+  }, []);
+
+  // Facet options derived from the loaded pools (distinct base tokens + price feeds).
+  const baseTokenOptions = useMemo((): string[] =>
+    [...new Set(pools.map(p => p.baseSymbol).filter(s => s && s !== "UNKNOWN"))].sort(),
```

ESLint

[error] Use new Set(pools.map(p => p.baseSymbol).filter(s => s && s !== "UNKNOWN")).toSorted() instead of copying and sorting

(e18e/prefer-array-to-sorted)